### ISSUE TITLE
"A picture is worth a thousand words"

### ISSUE THREAD

**Issue by mirdaki** [mirdaki/ping#6]
We need a logo (or I at least want one haha). It should be quick and dirty, just enough of something to help it stand out when sharing. Depending on how #5 ends up, it should be reminiscent of "Ping". Should be an SVG so things can easily scale.

**Comment by ChunkyProgrammer** [comment]
I know you plan on changing the name but I was thinking maybe a Ping pong paddle hitting a green ">" so it's like Ping and Matrix (I am not an artist)

![Pingtrix design](https://user-images.githubusercontent.com/78101139/107590568-695fee00-6bd6-11eb-8272-599c05af351f.png)

**Comment by mirdaki** [comment]
Oh I like the Ping Pong analogy. Depending on the name change, that is something we might end up referencing. Thanks for sharing @ChunkyProgrammer!

**Comment by ghost** [comment]
A variant of this?
![ping](https://user-images.githubusercontent.com/69373044/113037089-a5391d80-915a-11eb-8acc-4a7b3c9853f6.png)

**Comment by mirdaki** [comment]
Thank you for the suggestion @KrazyKirby99999! I like the connection to Matrix and the simplicity of it. That has given me an idea, give me a few and I'd like your thoughts on it

**Comment by mirdaki** [comment]
So this is more abstract draft (and I'm not at all sold on the colors haha), but what do you @ChunkyProgrammer  and @KrazyKirby99999 think about something like this? 
![image](https://user-images.githubusercontent.com/8786151/113494361-6083ee00-949c-11eb-991d-67d87bf82a20.png)


Also, color suggestions would be appreciated! Colors from the logo would be pulled into the UI at one point most likely

**Comment by ghost** [comment]
> So this is more abstract draft (and I'm not at all sold on the colors haha), but what do you @ChunkyProgrammer and @KrazyKirby99999 think about something like this?
> ![image](https://user-images.githubusercontent.com/8786151/113494361-6083ee00-949c-11eb-991d-67d87bf82a20.png)
> 
> Also, color suggestions would be appreciated! Colors from the logo would be pulled into the UI at one point most likely

I like the design.

I tried switching the colors up but I'm not a big fan of the color choices I made either (it just looks like a cookie or some kind of melon).  I would recommend converting the logo to a svg file (since you want to pull the colors from the UI) and using a program like Inkscape to play around with the colors until you find something that looks nice.

![matrix logo attempt](https://user-images.githubusercontent.com/78101139/113961977-e2557d80-97f4-11eb-9bce-4158c416df56.png)

**Comment by ghost** [comment]
@mirdaki , are there any specific colors that you have in mind?

**Comment by mirdaki** [comment]
> > So this is more abstract draft (and I'm not at all sold on the colors haha), but what do you @ChunkyProgrammer and @KrazyKirby99999 think about something like this?
> > ![image](https://user-images.githubusercontent.com/8786151/113494361-6083ee00-949c-11eb-991d-67d87bf82a20.png)
> > Also, color suggestions would be appreciated! Colors from the logo would be pulled into the UI at one point most likely
> 
> I like the design.
> 
> I tried switching the colors up but I'm not a big fan of the color choices I made either (it just looks like a cookie or some kind of melon). I would recommend converting the logo to a svg file (since you want to pull the colors from the UI) and using a program like Inkscape to play around with the colors until you find something that looks nice.
> 
> ![matrix logo attempt](https://user-images.githubusercontent.com/78101139/113961977-e2557d80-97f4-11eb-9bce-4158c416df56.png)

Agreed, I'll put together a cleaner version of it in Inkscape and share it with y'all!

**Comment by mirdaki** [comment]
> @mirdaki , are there any specific colors that you have in mind?

Not exactly. I want them to be a good enough combo that I can pull them into the app's UI to highlight or accent different UI elements

I also personally like teal as a nice color (which is why I originally used a shade of that), but I don't have strong feelings on keep that

It should also feel at home in a desktop taskbar or mobile launcher, though it standing out slightly can't hurt adoption 😁

**Comment by mirdaki** [comment]
OK @ChunkyProgrammer and @KrazyKirby99999, here is a cleaner version of the idea in an SVG format (though I have to upload it as a PNG here)

![Ping-Logo](https://user-images.githubusercontent.com/8786151/114330190-061b0b00-9af6-11eb-921b-cccfa432a193.png)

The shape is meant to be reminiscent of a vibration or alert animation when someone revives a message. It also happens to look a bit like a crosshair for the FPS crowd

I chose the blue-ish-purple as a base since it's in the same ballpark as Discord and Twitch and gives off a stable, but quality vibe (granted, not a color scientist haha, so that might just be me). The orange adds a bit of flair, is sort of complimentary to the purple, and isn't super common in the messaging world. It also feels a bit more Rust-y, as a nod to the programming language. I've added small borders around the elements to give it a bit more interest visually

I'd appreciate any critiques or suggestions 😁

**Comment by ghost** [comment]
What are the rgb values of each part?

**Comment by mirdaki** [comment]
The purple is 5340a5ff
It's border is 150070ff
The orange is f87047ff
It's border is 6d2f00ff

**Comment by ghost** [comment]
What if you use replace the orange with #C57711?

**Comment by mirdaki** [comment]
Sure! This is what that looks like

![Ping-Logo-2](https://user-images.githubusercontent.com/8786151/114650607-672e1480-9c97-11eb-87a2-aff6e8c68093.png)

It certainly meshes with the purple better!

**Comment by mirdaki** [comment]
I did show the logo to a couple friends. The sentiment was that the logo looks like a weird melon and they thought it wasn't super correlated with the name. Though it looks OK just buy itself.

They also both independently mentioned the ping-pong analogy when they first heard the name, so here is my shot at doing that. What do y'all think? I went for a bit more cartoon-y of a look this time

![Ping-Pong-Logo](https://user-images.githubusercontent.com/8786151/114650902-de63a880-9c97-11eb-9f95-8c39489c87ac.png)

**Comment by ghost** [comment]
What about making the paddle #5340a5ff, the handle #D68A15, centering the ball in the paddle, making the ball larger, and removing the star?

**Comment by mirdaki** [comment]
Sure! It's a good color combo. I do worry a little the ball makes it look sorta like an eye. Not quite sure how to change that.

![Ping-Pong-Logo](https://user-images.githubusercontent.com/8786151/114814737-a9278b00-9d69-11eb-9962-06bde4c4d22b.png)

**Comment by ghost** [comment]
What if the ball is replaced by the marks from https://user-images.githubusercontent.com/8786151/113494361-6083ee00-949c-11eb-991d-67d87bf82a20.png, but smaller and the same color as the ball?

**Comment by mirdaki** [comment]
OK, here are a few variants. I played with the color, outline, and shape combination. I'll label them to make it easier to refer to

A
![Ping-Pong-Logo-3](https://user-images.githubusercontent.com/8786151/115099898-d69d4180-9eed-11eb-89cc-049393901e5d.png)

B
![Ping-Pong-Logo-4](https://user-images.githubusercontent.com/8786151/115099923-051b1c80-9eee-11eb-8f03-28ff382bfa66.png)

C
![Ping-Pong-Logo-5](https://user-images.githubusercontent.com/8786151/115099927-12380b80-9eee-11eb-9689-1519103f782d.png)

D
![Ping-Pong-Logo-6](https://user-images.githubusercontent.com/8786151/115099931-16fcbf80-9eee-11eb-986c-c089b396d26a.png)

**Comment by ghost** [comment]
![Capture](https://user-images.githubusercontent.com/69373044/115115017-c1f59380-9f57-11eb-9fe2-88e98b464422.PNG)
This is what the icons look like on a windows desktop.  I would rank them B, A, C, D. The marks on B resemble a crosshair, and I think that it is more aesthetically pleasing.

**Comment by mirdaki** [comment]
Ah fantastic thought putting it on the desktop! My current desktop environment doesn't have well, desktop icons 😁, so I'd have to boot up my work machine to test that

I agree, B is the most pleasing of all of them. I did think the marks were a little small, so I made them larger. Also made color variants with the same white-grey and the gold-orange of the handle. What do you think?

E
![Ping-Pong-Logo-7](https://user-images.githubusercontent.com/8786151/115180045-8955d700-a089-11eb-86bb-f339ef51c9a8.png)

F
![Ping-Pong-Logo-8](https://user-images.githubusercontent.com/8786151/115180069-95da2f80-a089-11eb-9486-ac296be65947.png)

**Comment by jasikpark** [comment]
I like the combination of A and D the most 🙋 - A, but with a ping pong ball in the center, and definitely large impact strokes!

**Comment by mirdaki** [comment]
Here you go @jasikpark! Both with bigger impacts and the ping-pong ball, though varying impact types.

G
![Ping-Pong-Logo-9](https://user-images.githubusercontent.com/8786151/115184184-21f05500-a092-11eb-8cbc-8f48f93ae558.png)

H
![Ping-Pong-Logo-10](https://user-images.githubusercontent.com/8786151/115184216-32083480-a092-11eb-9851-ec2338abed5e.png)

**Comment by ghost** [comment]
![image](https://user-images.githubusercontent.com/69373044/115290024-c2895800-a118-11eb-91ed-30b33fc2cabd.png)
Out of these, E and G are the best.

**Comment by jasikpark** [comment]
G is exactly what I was going for, yeah :D

**Comment by mirdaki** [comment]
I've shown A-H to some folk who fall under the right audience for the project and they all preferred G. Are there any tweaks y'all can suggest for G? We can always revise the logo later, but I think we're close to done with the initial version I'll pull into the app

**Comment by ghost** [comment]
Maybe show the 4 diagonal lines at different sizes? Other than that, I do not see anything that might be benefit from being changed.

**Comment by ghost** [comment]
By different sizes, I mean the same sizes as each other, but different images. Each with its own set of sizes for the lines.

**Comment by mirdaki** [comment]
Sure! I've either increased the size or increased the size and moved the diagonals.

I
![Ping-Pong-Logo-11](https://user-images.githubusercontent.com/8786151/115998755-9c8f0800-a59d-11eb-8010-3c9382f22a35.png)

J
![Ping-Pong-Logo-12](https://user-images.githubusercontent.com/8786151/115998758-a284e900-a59d-11eb-9300-4574227bfbc6.png)

K
![Ping-Pong-Logo-13](https://user-images.githubusercontent.com/8786151/115998759-a6187000-a59d-11eb-984d-b52cce41d893.png)

**Comment by ghost** [comment]
It's difficult to tell them apart, but I slightly prefer K over the others.
![image](https://user-images.githubusercontent.com/69373044/116002692-89416400-a5c0-11eb-9e74-4e0e7dfe348f.png)
